### ISSUE TITLE
Scene description does not update tickmark range in regards to hand position immediately 

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/ratio-and-proportion#346]
**Test device**
Dell
**Operating System**
Win 10
**Browser**
Firefox
**Problem description**
For https://github.com/phetsims/QA/issues/601. Found with Firefox + NVDA and seen on a11y view.
When you have the tick marks on, the hands are described as being on or around a tick mark of a certain number. If you change the number of tick marks, than which tick mark a given hand is described as being on will change to match the new number. However, this only happens after you move one of the hands. If you go straight to the scene description it will use the old tick mark numbers.
**Steps to reproduce**
1. Go to the second screen with a screen reader and turn on tick marks.
2. Read through scene description and note tick mark numbers listed for hands
3. Go back to tick marks and change them to a new range.
4. Without changing hands, read scene description again
5. Use keyboard to move a hand back and forth
6. Read description again.

**Visuals**

https://user-images.githubusercontent.com/41024075/107087197-50ab9e80-67b8-11eb-874f-25ab7087028f.MOV

<img width="919" alt="updatewherehandsare" src="https://user-images.githubusercontent.com/41024075/107087372-936d7680-67b8-11eb-9ab9-8c7221277f79.png">

After movement:
<img width="899" alt="aftermovinghands" src="https://user-images.githubusercontent.com/41024075/107087326-86e91e00-67b8-11eb-8ce7-d08d880200a7.png">

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪Ratio and Proportion‬
URL: https://phet-dev.colorado.edu/html/ratio-and-proportion/1.0.0-rc.1/phet/ratio-and-proportion_all_phet.html
Version: 1.0.0-rc.1 2021-01-26 23:41:16 UTC
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:85.0) Gecko/20100101 Firefox/85.0
Language: en-US
Window: 1280x687
Pixel Ratio: 1.5/1
WebGL: WebGL 1.0
GLSL: WebGL GLSL ES 1.0
Vendor: Mozilla (Mozilla)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by KatieWoe** [comment]
This can also seemingly be fixed by adjusting the challenge ratio. Also note that this is in the play area of the scene description only from what I see so far.

**Comment by terracoda** [comment]
Good catch @KatieWoe! The dynamic description in the Play Area is not updating in the same way as the sceen summary.

**Comment by zepumph** [comment]
Nice indeed! Fixed above. I didn't see any others when inspecting what we are looking at in the screen summary. I'll put this into the branch.

**Comment by KatieWoe** [comment]
Looks good in rc.2 on both a11y view and NVDA with Firefox.
